docs/manuals/gui/viselements/part.md_template

```diff
@@ -78,10 +79,10 @@ in the Stylekit.
 
 ### Showing and hiding controls
 
-!!! example "Page content"
+A `part` can be hidden by setting the [*render*](#p-render) property to False:
```

Comment:
technically it is not hidden, it's not there

Reply:
I'll be more precise. Thanks.